org.webcamoid.Webcamoid.yml

```diff
@@ -0,0 +1,22 @@
+app-id: org.webcamoid.Webcamoid
```

Comment:
`webcamoid.org` doesnt resolve. Is this valid?

I see it's the one used upstream, but it has to be a valid domain under the project control.

Reply:
I would see `io.github.webcamoid.Webcamoid` being a proper id meeting the requirements.

Reply:
To quote the documentation:

> Each application should have a unique application ID, following the standard reverse-DNS schema. See [the Flatpak documentation](http://docs.flatpak.org/en/latest/conventions.html#application-ids) for more information on this. The Application ID should be a real URL of a domain that the app author has control over or where their app is hosted.

You can disagree. But that's the policy.

Reply:
This is also the future way you will be able to verify that you really are the maintainer of the app, and get a checkmark on the website.